docs/help/bundle.md

```diff
@@ -30,39 +30,25 @@ Before you build the bundle image generate the manifests and metadata:
 make IMAGE_TAG_BASE=<bundle image registry> OPERATOR_IMAGE_REPO=<operator image registry> OPERATOR_VERSION=<operator tag> bundle
 ```
 
-### Building the bundle image:
-
-```$xslt
-make IMAGE_TAG_BASE=<bundle image registry> bundle-build
-```
-The result image tag takes the form like
-```$xslt
-${IMAGE_TAG_BASE}-bundle:v0.0.1
-```
-Note: the version v0.0.1 is defined by VERSION variable in the Makefile
-
-To push the built bundle image
-
-```$xslt
-make IMAGE_TAG_BASE=<bundle image registry> bundle-push
-```
-
 ### Building the catalog image
 
-Now with the bundle image in place, build the catalog(index) iamge:
+#### Note:
+
+   - Users can utilize the official Artemis bundle images available at [quay.io/artemiscloud/activemq-artemis-operator-bundle](https://quay.io/repository/artemiscloud/activemq-artemis-operator-bundle?tab=tags&tag=latest) for building the catalog image.
+   - Alternatively, users can choose to use their own custom-built bundle images from their  [quay.io](https://quay.io/) repository.
```

Comment:
User can use repository of their choice. May be it's better to add `for example` before quay.io.

Reply:
Thanks!, i will modify the sentence.